### ISSUE TITLE
Empty blocks in the generated CSS

### ISSUE THREAD

**Issue by nono** [cozy/cozy-ui#1380]
In `dist/cozy-ui.min.css`, we can find several media queries with an empty block:

```css
@media (max-width:25rem) {}
@media (max-width:35rem) {}
@media (max-width:39rem) {}
@media (max-width:46rem) {}
@media (max-width:56rem) {}
@media (max-width:66rem) {}
```

**Comment by nono** [comment]
It looks like some of them comes from https://github.com/cozy/cozy-ui/blob/a5329ef8349675afae255d128847a30aac1775a3/stylus/components/modals.styl#L71-L77, but I don't understand why.

**Comment by Crash--** [comment]
I just took a look at this issue. We still have empty blocks in `cozy-ui.min.css` and `cozy-ui.utils.min.css`. 

The issue is that stylus is optimizing the generated CSS. So sometimes, when we arrive in a breakpoint() let's say in `small-screen` (https://github.com/cozy/cozy-ui/blob/master/stylus/settings/breakpoints.styl#L99-L101) then the `block` can be empty because the property is already applied. I haven't yet managed to get a minimal reproduction scenario

Current scenario:
- build.styl import settings/* tools/* animation/* and elements/default
- elements/defaults only require forms
- forms only have mixins and `$field-inline
    display flex
    align-items flex-start
    flex-direction row
    margin rem(8 0 8 24)

    +small-screen()
        flex-direction column
        margin-left 0
`

then we have an empty block generated. 

I tried to test if `block` was empty in the stylus, but didn't find something that worked. So I created a JS file to remove them.
